[BOP-1277] [Docs, MKE4] cadvisor (#124)
Mirantis/mke-docs: content/docs/release-notes/known-issues.md

   userID: 7668fdb9-a979-4645-b6cc-10985df77da6
   username: admin
3. Edit the ``hash`` field with the desired password hash.

## [BOP-1299] Disk Usage and Memory metrics are not shown correctly in the dashboard with disabled cAdvisor

> When cAdvisor is disabled, on the main page of the dashboard, "Disk Usage" and "Memory" metrics have zero values.

Reviewer: ```suggestion
When cAdvisor is disabled, the main page of the dashboard presents a `0` value for the **Disk Usage** and  **Memory** metrics.
```

Reviewer: As I have not seen the dashboard, I make an assumption in my previous comment about the actual value presented. If, in fact, the word 'zero' is presented instead, please make the appropriate edit.

Author: ![image](https://github.com/user-attachments/assets/e681e7f5-0487-4973-9ed0-783ae2bd2638)

this is what it looks like

Author: done

Reviewer: Thanks for the visual, @byDimasik. Having seen it, though, I would like to change the text to read:

"...presents 0% as the value for..."

Author: done

> When cAdvisor is disabled, the main page of the dashboard presents 0% as the value for the **Disk Usage** and  **Memory** metrics.

Reviewer 2: ```suggestion
When cAdvisor is disabled, the main page of the dashboard presents 0% as the value for the **Disk Usage** and **Memory** metrics.
```

**Workaround:**

> Enable cAdvisor in the config file and apply the config with `mkectl apply`

Reviewer: ```suggestion
Enable cAdvisor in the MKE configuration file and run `mkectl apply`.
```

Author: done

```yaml
monitoring:
   enableCAdvisor: true
```

> ## [BOP-1299] Max Used Disk and Max CPU labels are swapped in the dashboard

Reviewer: ```suggestion
## [BOP-1299] Max Used Disk and Max CPU labels are swapped in the cAdvisor dashboard
```

Author: I was referring to the MKE Dashboard. In general, the dashboard means the MKE Dashboard because this is the only dashboard we have. Do you want me to change it to "the MKE dashboard"?

Reviewer: Yes, let's say "MKE dashboard" here. There may be others out there who are as easily confused as me. :-)

Author: done

No workaround is available at this time.

> ## [BOP-1307] Prometheus can be access without authentication

Reviewer: ```suggestion
## [BOP-1307] Prometheus can be accessed without authentication
```

Author: done

> Anyone who knows MKE 4 URL can access Prometheus without authentication.

Reviewer: ```suggestion
Any party with knowledge of the MKE 4 URL can access Prometheus without authentication.
```

Author: done
